textalyze.js

```diff
@@ -14,10 +6,17 @@
 function itemCounts(array) {
   let counts = new Map();
 
-  // Your code here.
-  // Run 'npm test' to see what tests need to pass.
+  for (const a of array) {
+    counts.has(a) ? counts.set(a, counts.get(a) + 1) : counts.set(a, 1);
+  }
 
   return counts;
 }
 
+const elements = ["a", "a", "a", "b", "b", "c"];
```

Comment:
We'd like this file to be both included as a module and run via `npm start`.  Check out https://stackoverflow.com/questions/4981891/node-js-equivalent-of-pythons-if-name-main for how to do this.

You want

```javascript
if (require.main == module) {
  const array = ['a', 'a', 'a', ...];
}
```

Reply:
Make sure you're using ESlint.  I've set up the project to automatically enforce a style guide using AirBnb's recommended style.

You can set up your editor to do it automatically or you can manually run `npx eslint` to run the version included with the project on the command line.